runtimes/cloudformation/cfnpatcher/template.go

```diff
@@ -44,10 +44,12 @@ func fillContainerInfo(ctx context.Context, container *gabs.Container, parameter
 		if err != nil {
 			l.Warn().Str("image", image).Err(err).Msg("could not retrieve metadata from repository")
 		} else {
-			if repoInfo.Entrypoint != nil {
+			if repoInfo.Entrypoint != nil && !container.Exists("EntryPoint") {
+				l.Info().Str("image", container.S("Image").String()).Msgf("using default entrypoint %s", repoInfo.Entrypoint)
 				container.Set(repoInfo.Entrypoint, "EntryPoint")
 			}
-			if repoInfo.Command != nil {
+			if repoInfo.Command != nil && !container.Exists("Command") {
```

Comment:
Should we apply the default command when the template overrides the entrypoint? This must have the same behavior it had previously